scripts/install.sh

```diff
@@ -153,11 +160,22 @@ PLACEHOLDER_REGEX_BRANCH_ISSUE_NUMBER=$(printf -- "$PLACEHOLDER_REGEX_BRANCH_ISS
 PLACEHOLDER_REGEX_GIT_COMMIT_MESSAGES=$(printf -- "$PLACEHOLDER_REGEX_GIT_COMMIT_MESSAGES" | sed 's/\\/\\\\/g')
 printf -v PLACEHOLDER_REGEX_GIT_COMMIT_MESSAGES "%q" "$PLACEHOLDER_REGEX_GIT_COMMIT_MESSAGES"
 PLACEHOLDER_REGEX_GIT_COMMIT_MESSAGES=$(printf -- "$PLACEHOLDER_REGEX_GIT_COMMIT_MESSAGES" | sed 's/\//\\\//g')
+PLACEHOLDER_NEW_COMMIT_MESSAGE=$(printf -- "$PLACEHOLDER_NEW_COMMIT_MESSAGE" | sed 's/\//\\\//g')
+
+replace_placeholder_with_value () {
+	if [ $(uname -s) == "Linux" ]; then
+		sed -i "s/${1}/${2}/g" "$HOOK_FILE"
```

Comment:
Good catch! Is this important/relevant for all Linux?

Reply:
I’ve worked on other scripts that use the sed command and it’s been relevant for all Linux user.